server/database/data/data-generator.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable no-loop-func */
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-console */
+/* eslint-disable no-await-in-loop */
+const faker = require('faker');
+const path = require('path');
+const createCsvWriter = require('csv-writer').createObjectCsvWriter;
+const Utils = require('./data-generator-utils');
+
+const numOfHostels = 100;
+const numOfAuthors = 100;
+
+const hostelsFilePath = path.resolve(__dirname, 'hostels.csv');
+const reviewsFilePath = path.resolve(__dirname, 'reviews.csv');
+const authorsFilePath = path.resolve(__dirname, 'authors.csv');
+
+const updateThreshold = 5000;
+
+/* Data Generation */
+
+// takes in an integer and generates that many hostels
+async function generateHostels(num, callback) {
```

Comment:
All the three generator functions here share some parts of the code that could be reused easily.

Reply:
Thanks Elvin, I'll take a look at merging them.